### ISSUE TITLE
Unexpected error on beta

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/feathers-giveth#472]
```
0|feathers-beta  | 2021-04-07 20:20: error: Hook error: email validation failed: recipient: Path `recipient` is required. {"type":"FeathersError","name":"BadRequest","code":400,"className":"bad-request","errors":{"recipient":{"name":"ValidatorError","message":"Path `recipient` is required.","properties":{"message":"Path `recipient` is required.","type":"required","path":"recipient"},"kind":"required","path":"recipient"}},"stack":"BadRequest: email validation failed: recipient: Path `recipient` is required.\n    at new BadRequest (/home/deploy/feathers-giveth-beta/node_modules/feathers-mongoose/node_modules/@feathersjs/errors/lib/index.js:86:17)\n    at exports.errorHandler (/home/deploy/feathers-giveth-beta/node_modules/feathers-mongoose/lib/error-handler.js:38:36)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","timestamp":"2021-04-07T20:20:41.650Z"}
0|feathers-beta  | 2021-04-07 20:20: { BadRequest: email validation failed: recipient: Path `recipient` is required.
0|feathers-beta  | 2021-04-07 20:20:     at new BadRequest (/home/deploy/feathers-giveth-beta/node_modules/feathers-mongoose/node_modules/@feathersjs/errors/lib/index.js:86:17)
0|feathers-beta  | 2021-04-07 20:20:     at exports.errorHandler (/home/deploy/feathers-giveth-beta/node_modules/feathers-mongoose/lib/error-handler.js:38:36)
0|feathers-beta  | 2021-04-07 20:20:     at process._tickCallback (internal/process/next_tick.js:68:7)
0|feathers-beta  | 2021-04-07 20:20:   type: 'FeathersError',
0|feathers-beta  | 2021-04-07 20:20:   name: 'BadRequest',
0|feathers-beta  | 2021-04-07 20:20:   message:
0|feathers-beta  | 2021-04-07 20:20:    'email validation failed: recipient: Path `recipient` is required.',
0|feathers-beta  | 2021-04-07 20:20:   code: 400,
0|feathers-beta  | 2021-04-07 20:20:   className: 'bad-request',
0|feathers-beta  | 2021-04-07 20:20:   data: undefined,
0|feathers-beta  | 2021-04-07 20:20:   errors:
0|feathers-beta  | 2021-04-07 20:20:    { recipient:
0|feathers-beta  | 2021-04-07 20:20:       { ValidatorError: Path `recipient` is required.
0|feathers-beta  | 2021-04-07 20:20:           at validate (/home/deploy/feathers-giveth-beta/node_modules/mongoose/lib/schematype.js:1257:13)
0|feathers-beta  | 2021-04-07 20:20:           at /home/deploy/feathers-giveth-beta/node_modules/mongoose/lib/schematype.js:1240:7
0|feathers-beta  | 2021-04-07 20:20:           at Array.forEach (<anonymous>)
0|feathers-beta  | 2021-04-07 20:20:           at SchemaString.SchemaType.doValidate (/home/deploy/feathers-giveth-beta/node_modules/mongoose/lib/schematype.js:1185:14)
0|feathers-beta  | 2021-04-07 20:20:           at /home/deploy/feathers-giveth-beta/node_modules/mongoose/lib/document.js:2501:18
0|feathers-beta  | 2021-04-07 20:20:           at process._tickCallback (internal/process/next_tick.js:61:11)
0|feathers-beta  | 2021-04-07 20:20:         properties: [Object],
0|feathers-beta  | 2021-04-07 20:20:         kind: 'required',
0|feathers-beta  | 2021-04-07 20:20:         path: 'recipient',
0|feathers-beta  | 2021-04-07 20:20:         value: undefined,
0|feathers-beta  | 2021-04-07 20:20:         reason: undefined,
0|feathers-beta  | 2021-04-07 20:20:         [Symbol(mongoose:validatorError)]: true } },
0|feathers-beta  | 2021-04-07 20:20:   [Symbol(feathers-mongoose/error)]:
0|feathers-beta  | 2021-04-07 20:20:    { Error: email validation failed: recipient: Path `recipient` is required.
0|feathers-beta  | 2021-04-07 20:20:        at ValidationError.inspect (/home/deploy/feathers-giveth-beta/node_modules/mongoose/lib/error/validation.js:47:26)
0|feathers-beta  | 2021-04-07 20:20:        at formatValue (internal/util/inspect.js:491:31)
0|feathers-beta  | 2021-04-07 20:20:        at formatProperty (internal/util/inspect.js:1071:11)
0|feathers-beta  | 2021-04-07 20:20:        at formatRaw (internal/util/inspect.js:727:11)
0|feathers-beta  | 2021-04-07 20:20:        at formatValue (internal/util/inspect.js:509:10)
0|feathers-beta  | 2021-04-07 20:20:        at inspect (internal/util/inspect.js:189:10)
0|feathers-beta  | 2021-04-07 20:20:        at Object.formatWithOptions (util.js:84:12)
0|feathers-beta  | 2021-04-07 20:20:        at Console.(anonymous function) (console.js:203:15)
0|feathers-beta  | 2021-04-07 20:20:        at Console.warn (console.js:220:31)
0|feathers-beta  | 2021-04-07 20:20:        at process.NotifyFeature.onUnhandledRejection (/usr/lib/node_modules/pm2/node_modules/@pm2/io/build/main/features/notify.js:89:17)
0|feathers-beta  | 2021-04-07 20:20:        at process.emit (events.js:198:13)
0|feathers-beta  | 2021-04-07 20:20:        at emitPromiseRejectionWarnings (internal/process/promises.js:140:18)
0|feathers-beta  | 2021-04-07 20:20:        at process._tickCallback (internal/process/next_tick.js:69:34)
0|feathers-beta  | 2021-04-07 20:20:      errors: { recipient: [ValidatorError] },
0|feathers-beta  | 2021-04-07 20:20:      _message: 'email validation failed' } }
```

**Comment by mohammadranjbarz** [comment]
@aminlatifi 
I think this PR will fix it #455 
So after merging `develop` to `master` we probably don't have this issue

**Comment by aminlatifi** [comment]
Thanks @mohammadranjbarz
